src/zsl/interface/task.py

```diff
@@ -46,7 +46,7 @@ def fill_model_with_payload(data, obj):
         if not hasattr(obj, k):
             raise ModelConversionError(obj, k)
 
-        if is_scalar(v):
+        if is_scalar(v) or is_list(v):
             setattr(obj, k, v)
```

Comment:
This will not work, since if this is a list of maps, we should convert.

Reply:
Or list of other nested models. Please, add this functionality and then we could resolve the issue.

Reply:
True that it won''t be that easy. If the list contains nonscalar types, we need to instantiate them (because of line 46) but we can't do that in python since we don't know their type

Reply:
Just use hints and unite the processing, please.

Reply:
Let's use the same framework as we parse the app models from JSON using hints. This should work, right?